requirements.txt

```diff
@@ -0,0 +1,7 @@
+coverage < 4.0.0
+django >= 1.4, < 1.7
+djangorestframework < 3.0.0
```

Comment:
I think, we do.  But in DRF 3.0 implementation of serializers is different - so we'll have to do more stuff when we want it to work for 3.0.